lib/src/folding/parsers/python.dart

```diff
@@ -96,48 +49,88 @@ class PythonFoldableBlockParser extends AbstractFoldableBlockParser {
     return parser.blocks;
   }
 
-  int _getLastLine(
+  /// Compares two lists of blocks and combines them into one list
+  /// with priority to highlight blocks if blocks intersect.
+  ///
+  /// Lets consider this python code for example:
+  /// a = [     # 0
+  ///     1,    # 1
+  ///     2,    # 2
+  ///     3,    # 3
+  ///     4,]   # 4
+  /// Highlight block will return [0, 4].
+  /// Indent block will return [1, 3] block.
+  /// We need to skip [1, 3] block because it is inside [0, 4] block.
+  List<FoldableBlock> _combineBlocks(
     List<FoldableBlock> highlightBlocks,
```

Comment:
These arguments are better be named because the call looks like the order of the two does not matter but it matters a lot.